src/Gtk.jl

```diff
@@ -67,6 +70,59 @@ include("theme.jl")
 include("gio.jl")
 include("application.jl")
 
+function __init__()
+    # Set XDG_DATA_DIRS so that Gtk can find its icons and schemas
+    ENV["XDG_DATA_DIRS"] = join(filter(x -> x != nothing, [
+        dirname(adwaita_icons_dir),
+        dirname(hicolor_icons_dir),
+        joinpath(dirname(GTK3_jll.libgdk3_path), "..", "share"),
+        get(ENV, "XDG_DATA_DIRS", nothing),
+    ]), ":")
```

Comment:
Shouldn't the separator be a semicolon on Windows? @ianshmean can you please check if this fixes the issue you reported yesterday? As far as I can see, this variable is not currently set on Windows, so I'm not sure this will be sufficient...